lib/src/framework.dart

```diff
@@ -239,6 +239,10 @@ abstract class HookState<R, T extends Hook<R>> with Diagnosticable {
   @protected
   void didUpdateHook(T oldHook) {}
 
+  /// Called synchronously after the [HookWidget.build] method finished
+  @protected
+  void didBuild() {}
```

Comment:
I'm not fond of this API because 99.9% of hooks won't use this life-cycle

I'd like a way to do that, without having to iterate through **all** hooks as most of them don't care about this.

Reply:
What about having  a mixing `DidBuild` with that API ? We will be able to filter hooks that need postBuild and the once who don't.

Reply:
Another solution would be that the hook need to register to the didBuild event somehow. Let me know which one you prefer :) or if you have better ideas

Reply:
I'd prefer having something like "addListener" but for didBuild

Reply:
yeah that was my preferred solution, I'll look into this then :)